mhs/inbound/inbound/request/handler.py

```diff
@@ -33,14 +34,21 @@ def initialize(self, workflows: Dict[str, workflow.CommonWorkflow],
         self.party_id = party_id
         self.state_store = state_store
 
+    @time_request
     async def post(self):
-        logger.info('001', 'Inbound POST recieved: {request}', {'request': self.request})
+        logger.info('001', 'Inbound POST received: {request}', {'request': self.request})
+        request_message = None
```

Comment:
I don't think you need this line

Reply:
Pycharm complained at first saying request message might not have been assigned before it was used, doesn't seem to care now so I'll remove it